scripts/buildtools/commands/prepack.js

```diff
@@ -1,9 +1,34 @@
 const path = require('path');
 const fs = require('fs');
+const crypto = require('crypto');
 const shell = require('shelljs');
 const chalk = require('chalk');
 
 const BUILD_DIR = path.resolve(__dirname, '..', '..', '..', 'dist');
+const JS_DIR = path.join(BUILD_DIR, 'dist/js');
+const CSS_DIR = path.join(BUILD_DIR, 'dist/css');
+
+function generateSris () {
+  const jsFiles = fs.readdirSync(JS_DIR);
+  const cssFiles = fs.readdirSync(CSS_DIR);
+  return [
+    ...jsFiles.map(file => path.join(JS_DIR, file)), 
+    ...cssFiles.map(file => path.join(CSS_DIR, file))
+  ]
+  .filter(path => !path.endsWith('.map'))
+  .reduce((acc, path) => {
+    // get filename (key)
+    const parts = path.split('/');
+    const filename = parts[parts.length - 1];
```

Comment:
I think removing path and keeping only the filename opens the door to filename collisions and ambiguity.

in package.json

```json
{
  "sri": {
    "dist/dist/js/okta-sign-in.min.js": "sha256-xVeaYWjvwag0GjwomrpHsibzBJ6AZ6WSKzqx+8UPLPo="
  }
}
```

or separate json file at `dist/dist/sri.json`?

```json
{
  "js/okta-sign-in.min.js": "sha256-xVeaYWjvwag0GjwomrpHsibzBJ6AZ6WSKzqx+8UPLPo="
}
```